### PR TITLE
Add link to the Gutenberg issue tracker in 

### PR DESCRIPTION
**Pull request by SiobhyB**
To make it clearer where issues related to the editor's core functionality should be reported, this PR adds a link to the Gutenberg issue tracker to the templates found when adding a new issue.

<hr />

PR submission checklist:

- [x] I have considered adding unit tests where possible.
- [x] I have considered if this change warrants user-facing release notes [more info](https://github.com/wordpress-mobile/gutenberg-mobile/blob/trunk/Release-notes.md) and have added them to [RELEASE-NOTES.txt](https://github.com/wordpress-mobile/gutenberg-mobile/blob/trunk/RELEASE-NOTES.txt) if necessary.

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/config.yml [hunk 1]
```diff
@@ -0,0 +1,5 @@
+blank_issues_enabled: true
+contact_links:
+  - name: Report issues with core functionality to Gutenberg
```
Comment (dcalhoun) [inline .github/ISSUE_TEMPLATE/config.yml]: Definitely not necessary by any means, but we might consider retaining the pattern of the `name` being a noun and `about` describing the action for consistency with other templates. Please feel free to disregard this suggestion. 
```suggestion
  - name: Bug report (core Gutenberg)
```

File: .github/ISSUE_TEMPLATE/config.yml [hunk 2]
```diff
@@ -0,0 +1,5 @@
...
+    url: https://github.com/WordPress/gutenberg/issues/new?assignees=&labels=Mobile+App+-+i.e.+Android+or+iOS&projects=&template=Bug_report_mobile.md/
```
Comment (dcalhoun) [inline .github/ISSUE_TEMPLATE/config.yml]: Directly linking to the [Bug report (Mobile)](https://github.com/WordPress/gutenberg/issues/new?assignees=&labels=Mobile+App+-+i.e.+Android+or+iOS&projects=&template=Bug_report_mobile.md) template makes this link very convenient and easy to use. That said, I note that in the `about` field we mention "bug or enhancement request." For the latter, I believe the Gutenberg project generally expects contributors to use the [Feature request](https://github.com/WordPress/gutenberg/issues/new?assignees=&labels=%5BType%5D+Enhancement&projects=&template=Feature_request.md) template. 

Should we consider linking to the top-level [template page](https://github.com/WordPress/gutenberg/issues/new/choose) instead or do you feel this is a non-issue?

File: .github/ISSUE_TEMPLATE/config.yml [hunk 3]
```diff
@@ -0,0 +1,5 @@
...
+    about: The gutenberg-mobile issue tracker is intended for non-core issues, such as those related to Automattic-owned products, like Jetpack. If you have a bug or enhancement request related to the editor's core functionality, please report it to the Gutenberg repository.
```
Comment (dcalhoun) [inline .github/ISSUE_TEMPLATE/config.yml]: A small suggestion to align with the language found in the Gutenberg templates. Please feel free to disregard. 
```suggestion
    about: The gutenberg-mobile issue tracker is intended for non-core issues, such as those related to Automattic-owned products, like Jetpack. For bugs or enhancement requests related to the editor's core functionality, please report it to the Gutenberg repository.
```

**Review by dcalhoun — APPROVED** [review]
Thank you! I think this is helpful. I hope others find it helpful as well. 👏🏻 

I left a few comments that are most definitely nitpicks. Please feel free to disregard them entirely if you believe them to be non-issues or unhelpful.

File: .github/ISSUE_TEMPLATE/config.yml (hunk 2 above)
Comment (SiobhyB) [reply .github/ISSUE_TEMPLATE/config.yml]: That reasoning for updating to the top-level page makes a lot of sense to me, and I've updated in https://github.com/wordpress-mobile/gutenberg-mobile/pull/6484/commits/71e14ac499e0589724b415a4c382d251a1d05885. Thank you for that as well as your other thoughtful suggestions, David! 🙇‍♀️

**Pull request merged**